[2021-10-16 13:05:44,812] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_0 scheduled__2021-10-16T12:24:00+00:00 [queued]>
[2021-10-16 13:05:44,958] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_0 scheduled__2021-10-16T12:24:00+00:00 [queued]>
[2021-10-16 13:05:44,958] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:05:44,958] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-16 13:05:44,958] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:05:45,076] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): get_gross_year_0> on 2021-10-16 12:24:00+00:00
[2021-10-16 13:05:45,122] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'get_gross_year_0', 'scheduled__2021-10-16T12:24:00+00:00', '--job-id', '482', '--raw', '--subdir', 'DAGS_FOLDER/fourth_dag.py', '--cfg-path', '/tmp/tmpyo23swrk', '--error-file', '/tmp/tmp6a6_f6jr']
[2021-10-16 13:05:45,087] {standard_task_runner.py:52} INFO - Started process 930 to run task
[2021-10-16 13:05:45,157] {standard_task_runner.py:77} INFO - Job 482: Subtask get_gross_year_0
[2021-10-16 13:05:45,413] {logging_mixin.py:109} INFO - Running <TaskInstance: fourth_dag.get_gross_year_0 scheduled__2021-10-16T12:24:00+00:00 [running]> on host e03fb9e01fd8
[2021-10-16 13:05:45,826] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fourth_dag
AIRFLOW_CTX_TASK_ID=get_gross_year_0
AIRFLOW_CTX_EXECUTION_DATE=2021-10-16T12:24:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-16T12:24:00+00:00
[2021-10-16 13:05:45,852] {python.py:152} INFO - Done. Returned value was: ['$29.37 ', '2007\n']
[2021-10-16 13:05:46,017] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=get_gross_year_0, execution_date=20211016T122400, start_date=20211016T130544, end_date=20211016T130546
[2021-10-16 13:05:46,257] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-16 13:05:46,445] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
